fuzzers/aflplusplus/fuzzer.py

```diff
@@ -173,8 +167,8 @@ def fuzz(input_corpus, output_corpus, target_binary, flags=tuple()):
     # os.environ['AFL_PRELOAD'] = '/afl/libdislocator.so'
 
     flags = list(flags)
-    if not flags or not flags[0] == '-Q' and '-p' not in flags:
-        flags += ['-p', 'seek']
+    #if not flags or not flags[0] == '-Q' and '-p' not in flags:
```

Comment:
Just to confirm that this is intentional.

Reply:
seek is the new default so this is not needed atm.
however as a bug affected all schedules I have to rerun a schedule test and that might result in a new default :)